packages/mobile/components/ArticleContent.tsx

```diff
@@ -0,0 +1,271 @@
+import * as React from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import { useState, useEffect } from "react";
+
+import styled from "styled-components";
+import { ActivityIndicator } from "react-native-paper";
+import { View, Text, SafeAreaView, Button } from "react-native";
+import moment from "moment";
+import Breadcrumbs from "./Breadcrumbs";
+import RichTextEditor from "./RichTextEditor";
```

Comment:
Let's not copy the name of RichTextEditor since that's already taken, how does ArticleEditor sound?